### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [chikaram/coder-school-ml01#1]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You accomplished fully the assignment. What a great start !
- Your solution is comprehensive and clean. Awesome !

## To sum up:
- Overall, excellent work! You are demonstrating that you have a great understanding the material and doing a great job of applying it. Keep it up!
